src/config/cache.config.ts

```diff
@@ -3,7 +3,7 @@ import { VOLTA_CHAIN_ID } from "../utils/constants";
 
 const cacheConfig: Record<number, CacheServerClientOptions> = {
     [VOLTA_CHAIN_ID]: {
-        url: "https://volta-identitycache.energyweb.org/v1",
+        url: "https://identitycache-dev.energyweb.org/v1/",
```

Comment:
Let's make this change in another PR
```suggestion
        url: "https://volta-identitycache.energyweb.org/v1/",
```